### ISSUE TITLE
训练icdar2015识别，acc一直为0，loss也没有怎么变化，请问可能是什么原因

### ISSUE THREAD

**Issue by Luis-lsy** [PaddlePaddle/PaddleOCR#3602]
[2021/08/08 22:15:57] root INFO: epoch: [72/72], iter: 10620, lr: 0.000100, loss: 14.480381, acc: 0.000000, norm_edit_dis: 0.230483, reader_cost: 0.00006 s, batch_cost: 0.88902 s, samples: 300, ips: 33.74485
[2021/08/08 22:16:06] root INFO: epoch: [72/72], iter: 10630, lr: 0.000100, loss: 14.154110, acc: 0.000000, norm_edit_dis: 0.224001, reader_cost: 0.00006 s, batch_cost: 0.89093 s, samples: 300, ips: 33.67277
[2021/08/08 22:16:15] root INFO: epoch: [72/72], iter: 10640, lr: 0.000100, loss: 13.778421, acc: 0.000000, norm_edit_dis: 0.237652, reader_cost: 0.00006 s, batch_cost: 0.89231 s, samples: 300, ips: 33.62061
[2021/08/08 22:16:24] root INFO: epoch: [72/72], iter: 10650, lr: 0.000100, loss: 14.129663, acc: 0.000000, norm_edit_dis: 0.240251, reader_cost: 0.00007 s, batch_cost: 0.88726 s, samples: 300, ips: 33.81186
[2021/08/08 22:16:28] root INFO: save model in ./output/rec/ic15/latest
[2021/08/08 22:16:28] root INFO: best metric, acc: 0.00048146364949446316, norm_edit_dis: 0.14544754181249087, fps: 130.10372712733766, best_epoch: 68

Global:
  use_gpu: false
  epoch_num: 72
  log_smooth_window: 20
  print_batch_step: 10
  save_model_dir: ./output/rec/ic15/
  save_epoch_step: 10
  # evaluation is run every 2000 iterations
  eval_batch_step: [0, 2000]
  # if pretrained_model is saved in static mode, load_static_weights must set to True
  cal_metric_during_train: True
  pretrained_model:
  checkpoints:
  save_inference_dir:
  use_visualdl: False
  infer_img: doc/imgs_words_en/word_10.png
  # for data or label process
  character_dict_path: ppocr/utils/ic15_dict.txt
  character_type: ch
  max_text_length: 25
  infer_mode: False
  use_space_char: True

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    learning_rate: 0.0001
  regularizer:
    name: 'L2'
    factor: 0

Architecture:
  model_type: rec
  algorithm: CRNN
  Transform:
  Backbone:
    name: MobileNetV3
    scale: 0.5
    model_name: large
  Neck:
    name: SequenceEncoder
    encoder_type: rnn
    hidden_size: 96
  Head:
    name: CTCHead
    fc_decay: 0

Loss:
  name: CTCLoss

PostProcess:
  name: CTCLabelDecode

Metric:
  name: RecMetric
  main_indicator: acc

Train:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data/ic15_data/
    label_file_list: ["./train_data/ic15_data/rec_gt_train.txt"]
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - CTCLabelEncode: # Class handling label
      - RecResizeImg:
          image_shape: [3, 32, 100]
      - KeepKeys:
          keep_keys: ['image', 'label', 'length'] # dataloader will return list in this order
  loader:
    shuffle: True
    batch_size_per_card: 30
    drop_last: True
    num_workers: 0
    use_shared_memory: False

Eval:
  dataset:
    name: SimpleDataSet
    data_dir: ./train_data/ic15_data/
    label_file_list: ["./train_data/ic15_data/rec_gt_test.txt"]
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - CTCLabelEncode: # Class handling label
      - RecResizeImg:
          image_shape: [3, 32, 100]
      - KeepKeys:
          keep_keys: ['image', 'label', 'length'] # dataloader will return list in this order
  loader:
    shuffle: False
    drop_last: False
    batch_size_per_card: 30
    num_workers: 0
    use_shared_memory: False

**Comment by Jsooooooo** [comment]
我的也是MobileNet large训练不了，但是small是可以训练的
